mod/cornercutter/ModLoader/CutterConfig.cs

```diff
@@ -61,6 +62,7 @@ public void LoadGlobalSettings()
             string settingsLocation = Path.GetFullPath(Path.Combine(cornercutterFolder, "settings.json"));
             GlobalInfoDTO settings = Reader.ReadGlobalInfo(settingsLocation);
             GlobalOptions = settings.GlobalOptions;
+            HasCurrentMod = (settings.CurrentModFilename != null);

```

Comment:
Is this any particular reason this is in brackets?

Reply:
for fun